doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_topology.rst

```diff
@@ -0,0 +1,478 @@
+
+It is worth to consider assigning the same Manifest Class ID for Radio Local Manifest for both products, for example::
+
+   uuid5(acme_vid, 'nRF54H20_radio')
+
+and two distinct Manifest Class ID values for Application Local Manifest, for example::
+
+    uuid5(acme_vid, 'nRF54H20_light_bulb_app')
+
+    uuid5(acme_vid, 'nRF54H20_roller_shutter_app')
+
+.. figure:: ../../../../../../../modules/doc-internal/sw_arch/suit/images/suit_acme_manifests.png
+
+   Manifest hierarchies for ACME Corp devices.
+
+This approach allows you to share the same radio domain update package (executable binaries and SUIT manifest) across both products.
+Additionally, if an application domain update package intended for the light bulb is accidentally delivered to the roller shutter motor, the motor will reject it due to the unrecognized (unsupported) Manifest Class ID.
+
+Downgrade prevention policy
+===========================
+
+Product requirements can impose different downgrade prevention policies for different manifests.
+The following policy values are supported:
+
+* ``downgrade_prevention_disabled``
+* ``downgrade_prevention_enabled``
+
+How downgrade prevention works
+------------------------------
+
+The ``suit-manifest-sequence-number`` (an element of the SUIT manifest) is a monotonically increasing anti-rollback counter.
+As part of the update candidate evaluation process, the suit-manifest-sequence-number of the candidate manifests is compared with the installed counterpart.
+The result of this comparison, along with the associated downgrade prevention policy, determines whether a candidate is accepted or rejected.
+
+Assuming ``downgrade_prevention_enabled`` is enabled, a candidate manifest will be accepted only if its ``suit-manifest-sequence-number`` is not lower than the ``suit-manifest-sequence-number`` of the installed manifest.
+
+Independent updateability policy
+================================
+
+This policy allows the expression of whether a candidate manifest of a specific Manifest Class ID can be updated independently of its parent.
+In some cases, this can be the intended behavior, while in others, the opposite can be desired.
+
+The following policy values are supported:
+
+* ``independent_update_allowed``
+* ``independent_update_denied``
+
+How independent updateability policy works
+------------------------------------------
+
+The system is designed to allow the expression of an update package as a candidate manifest along with its dependencies (candidate, child manifests).
+Considering the simplest case, an update package composed of just one manifest (for example, a Local App Manifest), the policy ``independent_update_denied`` associated with the Local App Manifest would block the update.
+
+This be considered intended behavior because images for the App Core (controlled by the Local App Manifest) can rely on functionality provided by images controlled by other manifests, such as the radio manifest.
```

Comment:
```suggestion
This be considered intended behavior because images for the Application Core (controlled by the Local App Manifest) can rely on functionality provided by images controlled by other manifests, such as the radio manifest.
```